src/components/user/user.service.ts

```diff
@@ -497,6 +497,9 @@ export class UserService {
       },
       session
     );
+    if (user.canEdit === null) {
+      user.canEdit = false;
+    }
```

Comment:
Would you mind doing this as
```tsx
canCreate: user.canEdit ?? false,
```

It's shorter and it doesn't mutate the `user` object